### ISSUE TITLE
Do we need a discord channel or something?

### ISSUE THREAD

**Issue by crlf0710** [crlf0710/tectonic#158]
Feel it's a little strange to post updates here, shall we create a discord channel or something?

Recent status update: I have just finished reading the first pass of the book tex the program. Last week I've started trying to write a rust crate to deal with original WEB files (literate programming) and hope that can become a web2rust component acting just like the web2c component in the texlive suite. After that maybe we can rely on that generating better rust code for the tex.web and xetex.web files.

Sorry for didn't spend enough time on this in the last three weeks. If you've got something specific for me (like a problematic pdf file or something) you can open ping me directly to investigate on it.

**Comment by cormacrelf** [comment]
I would recommend Zulip over Discord any day. The threading is incredibly good, and it has worked very well [for the Salsa project](https://salsa.zulipchat.com). And you can log in with GitHub.

**Comment by cormacrelf** [comment]
There is already a tectonic Zulip apparently. Invite only. https://tectonic.zulipchat.com/login/

**Comment by crlf0710** [comment]
@cormacrelf yes, i registered that. Now i've turned off the need invitation settings, and everybody can join now.

**Comment by crlf0710** [comment]
Now just need an update to README.
